http/src/main/java/io/crate/plugin/HttpTransportPlugin.java

```diff
@@ -130,7 +128,7 @@ public Settings additionalSettings() {
 
     @Override
     public UnaryOperator<RestHandler> getRestHandlerWrapper(ThreadContext threadContext) {
-        return restHandler -> new CrateRestMainAction.RestFilter(settings, restHandler);
+        return CrateRestMainAction.RestFilter::new;
```

Comment:
cant we remove this method override completely (and also the RestFilter)? looks like the filter is not doing anything anymore.